### PR TITLE
[Merged by Bors] - Release 0.6.0

### PR DESCRIPTION
**Pull request by cart**

### CODE REVIEW

**Review by ickk — APPROVED** [review]
ship it

File: crates/bevy_app/Cargo.toml [hunk 1]
```diff
@@ -15,10 +15,10 @@ default = ["bevy_reflect"]
 
 [dependencies]
 # bevy
-bevy_derive = { path = "../bevy_derive", version = "0.5.0" }
```
[the remark this answers is not in the document]
Comment (ickk) [reply crates/bevy_app/Cargo.toml:18]: I expect it was probably a search & replace operation, not really a big deal

[the remark this answers is not in the document]
Comment (cart) [reply crates/bevy_app/Cargo.toml:18]: Haha it was either `cargo release` or the find and replace I did after that :)

**Pull request closed without merging**